rules/typescript.js

```diff
@@ -36,14 +36,17 @@ const typescriptRules = {
     {
       selector: 'default',
       format: ['camelCase'],
+      leadingUnderscore: 'allow',
```

Comment:
Should we only allow this for functional arguments? Such as:

```js
onChange = (_ev) => {}
```

Reply:
im running into it with this: `const [_rows, _groups] = getSortedQuestions(isAdmin, questions)`. i think we could try this out and restrict as we go if we notice weird behavior

Reply:
Oh right, that's a good use-case. Optional suggestion: we only add this for 'variable' and 'argument' (not sure if argument is one of the selector types)

Reply:
And private variables such as `_index` is probably valid IMO